src/growth_region.h

```diff
@@ -74,20 +74,24 @@ class GrowthRegion : public cyclus::Region {
   #pragma cyclus var {"tooltip": "commodity in demand", \
                       "doc": "name of the commodity experiencing a " \
                              "growth in demand", \
+                      "uilabel": "Growth Commodity", \
```

Comment:
No - uitype is how cycic determines if certain fields represent commodities or prototypes or nuclides - because those need to get special treatment - e.g. commodities are used to connect prototypes together graphically.  Also uitype is used by cycpp so it knows when to pass a value that represents a nuclide through ``pyne::nucname::id`` to compute the canonical nuclide id.